modules/ROOT/pages/connected-apps-permissions.adoc

```diff
@@ -0,0 +1,80 @@
+= Connected Apps Permissions by User
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Connected Apps can be used by organization administrators, developers, and end users; each of these roles has different privileges when connecting applications,
```

Comment:
please standardize on one term for connecting an application: Using / connecting / authorizing.